tests/conftest.py

```diff
@@ -35,13 +117,13 @@ def pytest_addoption(parser):
     parser.addoption(
         "--stress-tested-api",
         action="store",
-        default=f"http://{LOCALHOST}:{NODES['1']['api_port']}",
+        default=f"http://{LOCALHOST}:{NODES[0].api_port}",
         help="The API towards which the stress test is performed",
     )
     parser.addoption(
         "--stress-tested-api-token",
         action="store",
-        default=DEFAULT_API_TOKEN,
+        default=API_TOKEN,
```

Comment:
This will also break the commented out test_security and test_websocket_api files.

Reply:
I already changed the name of the token accordingly in all files.